CONTRIBUTING.md

```diff
@@ -150,6 +146,43 @@ For this project treat `deploy/preview` brach as `main`.
         pg_restore -U postgres -d www www_backup_latest
         ```
 
+    3.  Check extensions installed in the `www` database.
+
+        ``` bash
+        psql -U postgres
+        ```
+
+        ``` text
+        \c www
+        ```
+
+        ``` text
+        \dx
+        ```
+
+        ``` text
+                                                    List of installed extensions
+                Name        | Version |   Schema   |                              Description
+        --------------------+---------+------------+------------------------------------------------------------------------
+         adminpack          | 2.1     | pg_catalog | administrative functions for PostgreSQL
+         amcheck            | 1.3     | public     | functions for verifying relation integrity
+         bloom              | 1.0     | public     | bloom access method - signature file based index
+         btree_gin          | 1.3     | public     | support for indexing common datatypes in GIN
+         btree_gist         | 1.6     | public     | support for indexing common datatypes in GiST
+         citext             | 1.6     | public     | data type for case-insensitive character strings
+         fuzzystrmatch      | 1.1     | public     | determine similarities and distance between strings
+         pageinspect        | 1.9     | public     | inspect the contents of database pages at a low level
+         pg_buffercache     | 1.3     | public     | examine the shared buffer cache
+         pg_freespacemap    | 1.2     | public     | examine the free space map (FSM)
+         pg_stat_statements | 1.9     | public     | track planning and execution statistics of all SQL statements executed
+         pg_trgm            | 1.6     | public     | text similarity measurement and index searching based on trigrams
+         pg_visibility      | 1.2     | public     | examine the visibility map (VM) and page-level visibility info
+         pgrowlocks         | 1.2     | public     | show row-level locking information
+         pgstattuple        | 1.5     | public     | show tuple-level statistics
+         plpgsql            | 1.0     | pg_catalog | PL/pgSQL procedural language
+        (16 rows)
+        ```
```

Comment:
The new snapshots will not have admin pack extensions. Not sure it's worth listing them.

Reply:
I'll add a comment regarding that.